clients/js/src/ChromaClient.ts

```diff
@@ -41,14 +41,8 @@ export class ChromaClient {
         fetchOptions,
         auth,
         tenant = DEFAULT_TENANT,
-        database = DEFAULT_DATABASE
-    }: {
-        path?: string,
-        fetchOptions?: RequestInit,
-        auth?: AuthOptions,
-        tenant?: string,
-        database?: string,
-    } = {}) {
+        database = DEFAULT_DATABASE,
+    }: ChromaClientParams = {}) {
```

Comment:
i used this PR to also move TS types out of inline, that way they can exported and consumed by other TS/JS envs